### ISSUE TITLE
Customizing TabbedPage

### ISSUE THREAD

**Issue by Chandraganesh** [rid00z/FreshMvvm#223]
Hi,

How can I write a custom renderer for TabbedPage using FreshMVVM.

**Comment by Codeadmire** [comment]
Please I am also want to know. Please help me

**Comment by Codeadmire** [comment]
Any updates regarding how to use custom render for TabbedPage for TopTabbedPage in IOS and BottomTabbedPage in Android ?
If I am using Customrender it will not overrride with FreshMVVM. I am getting same result based on Platform.
Please anyone provide and suggestion to over come this issue.

**Comment by maherzaidoune** [comment]
you can have a look at these project https://github.com/sthewissen/KickassUI.Spotify , it uses BottomNavigationBarXF  to render a buttom tabbed Page

**Comment by libin85** [comment]
Yes, you can. If you are after a bottom tabbed navigation, you may also use https://github.com/NAXAM/bottomtabbedpage-xamarin-forms. Alternatively , if you are jusing Xamarin Forms 3.1, it supports out of box now

**Comment by claudioluciano** [comment]
Just add `this.On<Android>().SetToolbarPlacement(ToolbarPlacement.Bottom);` on the `TabbedPage`
you will need Xamarin Forms 3.0

**Comment by seansparkman** [comment]
Where do you execute that?

**Comment by ForteUnited** [comment]
> Where do you execute that?

I think you can call it from anywhere really. I'm calling it right in my App.xaml.cs after I build out my TabbedNavigationContainer for the main navigation. Works just fine.
